src/test/java/com/test/carina/demo/demoBlazeTest.java

```diff
@@ -0,0 +1,75 @@
+package com.test.carina.demo;
+
+import com.qaprosoft.carina.core.foundation.IAbstractTest;
+import com.test.carina.demo.gui.demoblaze.CartPage;
+import com.test.carina.demo.gui.demoblaze.HomePage;
+import com.test.carina.demo.gui.demoblaze.ProductDescPage;
+import com.zebrunner.carina.core.registrar.tag.Priority;
+import com.zebrunner.carina.core.registrar.tag.TestPriority;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+import java.util.List;
+
+public class demoBlazeTest implements IAbstractTest {
```

Comment:
classes should start with capital letter
```suggestion
public class DemoBlazeTest implements IAbstractTest {
```